fe/fe-core/src/main/java/org/apache/doris/metric/MetricRepo.java

```diff
@@ -492,7 +494,9 @@ public static synchronized String getMetric(MetricVisitor visitor) {
 
         visitor.setMetricNumber(PALO_METRIC_REGISTER.getPaloMetrics().size());
         // doris metrics
-        for (Metric metric : PALO_METRIC_REGISTER.getPaloMetrics()) {
+        List<Metric> paloMetrics = new ArrayList<>(PALO_METRIC_REGISTER.getPaloMetrics());
+        paloMetrics.sort(Comparator.comparing(Metric::getName));
```

Comment:
How about replacing the `paloMetrics` in PALO_METRIC_REGISTER to a PriorityQueue, and the metric can be sorted when added.
So that we don't need to sort the metric every time we visit it.

Reply:
Yeah, that's more reasonable!